src/file_picker.js

```diff
@@ -1,16 +1,16 @@
-import Vue from 'vue';
```

Comment:
I think we should standardize using strict. I mean, using semicolons. It may be OK here in this one case, but I think out of habit and consistency we should use them all over the place.  I don't to have to debug an issue for hours only to find out the statement isn't what I _thought_ it was because of newlines.

Reply:
I now realize I'm conflating `use strict` and using semicolons. We should do both, but in this review/change specifically we should continue to use semi colons.